### PR TITLE
Refactor Lookup Client

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Refactor Lookup Client
Add some extra tests

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [:D] My code follows the style guidelines of this project
- [:D] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [:D] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/ServiceFees.java [hunk 1]
```diff
@@ -13,32 +13,12 @@
...
-            @JsonProperty("entUserId") String entUserId,
-            @JsonProperty("serviceTypeCd") String serviceTypeCd,
-            @JsonProperty("effectiveDt") String effectiveDt,
-            @JsonProperty("updUserId") String updUserId,
-            @JsonProperty("entDtm") String entDtm,
-            @JsonProperty("expiryDt") String expiryDt) {
-
-        this.udpDtm = udpDtm;
+            @JsonProperty("serviceTypeCd") String serviceTypeCd) {
         this.feeAmt = feeAmt;
```
Comment (alexjoybc) [inline src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/ServiceFees.java:18]: can we change to `feeAmount`

File: src/backend/libs/efiling-cso-starter/src/main/java/ca/bc/gov/open/jag/efilingcsostarter/CsoLookupServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    public CsoLookupServiceImpl(LookupFacadeBean lookupFacadeItf) {
+        this.lookupFacadeItf = lookupFacadeItf;
+    }
+
+    @Override
+    public ServiceFees getServiceFee(String serviceId)  {
+
+        // NOTE- "DCFL" is the only string that will work here until we get our service types setup
+        // TODO: throw exception when service fee is null
```
Comment (alexjoybc) [inline src/backend/libs/efiling-cso-starter/src/main/java/ca/bc/gov/open/jag/efilingcsostarter/CsoLookupServiceImpl.java:26]: remove the TODO

File: src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/ServiceFees.java (hunk 1 above)
Comment (TayGov) [reply src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/ServiceFees.java]: Updated

File: src/backend/libs/efiling-cso-starter/src/main/java/ca/bc/gov/open/jag/efilingcsostarter/CsoLookupServiceImpl.java (hunk 2 above)
Comment (TayGov) [reply src/backend/libs/efiling-cso-starter/src/main/java/ca/bc/gov/open/jag/efilingcsostarter/CsoLookupServiceImpl.java]: Done

**Pull request merged**